src/g2_gbytesc.f

```diff
@@ -63,24 +88,25 @@ SUBROUTINE G2_GBYTESC(IN,IOUT,ISKIP,NBYTE,NSKIP,N)
       RETURN
       END                                                                  
 
+!>    This subrountine is to put arbitrary size values into a packed bit
+!>    string, taking the low order bits from each value in the unpacked
+!>    array with skip and interation options.
+!>
+!>    @param[in] IN unpacked array input
+!>    @param[out] OUT packed array output
+!>    @param[in] ISKIP initial number of bits to skip
+!>    @param[in] NBYTE number of bits to pack
+!>    @param[in] NSKIP additional number of bits to skip on each iteration
+!>    @param[in] N number of iterations
```

Comment:
Reorder these @param tags so the order is the same as the order of the parameters in the subroutine declaration.

Reply:
Okay, If author is unknown. I can add it based on the logical relationship. That should still be Stephen.

Reply:
If you can make a good guess, do so.

If you can confirm with Stephen (or whomever you guess), do so.

If you can't even guess, put "NOAA programmer". This is a worst case scenario.

@DusanJovic-NOAA here's a great example of GitHub not providing the authorship info we want to have, and why NCEPLIBS code now required full attribution in the comments. This came up a lot in the UFS_UTILS doxygenation project - fortunately, there are still employees working here who remember who wrote what. Once those employees leave, then that information would be lost, had we not captured it in the documentation.